packages/main/src/ListItem.js

```diff
@@ -214,13 +218,14 @@ class ListItem extends ListItemBase {
 	_onkeydown(event) {
 		super._onkeydown(event);
 
-		const itemActive = this.type === ListItemType.Active;
+		const itemActive = this.type === ListItemType.Active,
+			  navigationItem = this.type === ListItemType.Navigation;
```

Comment:
itemNavigated would be more consistent with "itemActive" and you can reuse the new getter "typeNavigation", which checks the same:
this.type === ListItemType.Navigation;